[2024-05-28T20:58:16.787+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-28T20:58:16.951+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_dag_1.python_task manual__2024-05-28T20:51:41.175049+00:00 [queued]>
[2024-05-28T20:58:17.003+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_dag_1.python_task manual__2024-05-28T20:51:41.175049+00:00 [queued]>
[2024-05-28T20:58:17.007+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 4
[2024-05-28T20:58:17.048+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): python_task> on 2024-05-28 20:51:41.175049+00:00
[2024-05-28T20:58:17.081+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4586) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-28T20:58:17.098+0000] {standard_task_runner.py:63} INFO - Started process 4597 to run task
[2024-05-28T20:58:17.100+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'example_dag_1', 'python_task', 'manual__2024-05-28T20:51:41.175049+00:00', '--job-id', '2491', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpahei7fk8']
[2024-05-28T20:58:17.130+0000] {standard_task_runner.py:91} INFO - Job 2491: Subtask python_task
[2024-05-28T20:58:17.416+0000] {task_command.py:426} INFO - Running <TaskInstance: example_dag_1.python_task manual__2024-05-28T20:51:41.175049+00:00 [running]> on host 6ecac41592ab
[2024-05-28T20:58:18.068+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='gurlenko.13032001@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_dag_1' AIRFLOW_CTX_TASK_ID='python_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-28T20:51:41.175049+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-28T20:51:41.175049+00:00'
[2024-05-28T20:58:18.080+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-28T20:58:18.147+0000] {base.py:84} INFO - Using connection ID 'coin_conn' for task execution.
[2024-05-28T20:58:18.298+0000] {logging_mixin.py:188} INFO - Current rate 68283.2
[2024-05-28T20:58:18.310+0000] {python.py:237} INFO - Done. Returned value was: 68283.2
[2024-05-28T20:58:18.316+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-28T20:58:18.463+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=example_dag_1, task_id=python_task, run_id=manual__2024-05-28T20:51:41.175049+00:00, execution_date=20240528T205141, start_date=20240528T205816, end_date=20240528T205818
[2024-05-28T20:58:18.566+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-28T20:58:18.725+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-28T20:58:18.737+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
